CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 ChangeLog
 =========
 
+5.1.0 (2020-01-31)
+-------------------------
+
+* Added support for PHP 7.4, dropped support for PHP 7.0 (@phil-davis)
```

Comment:
@phil-davis looking at the current .travis.yml I am wondering why we dont have builds for 7.1?

Reply:
#142 set the PHP version in `composer.json` to 7.2 or greater.
That dropped PHP 7.1 support in this repo.

Hmmm - `dav` depends on `http`. So if we drop PHP 7.1 here in `http` then that will force `dav` to drop PHP 7.1. But probably we are not forced to do that right now. We could put back PHP 7.1 support here in `http`

(e.g. ownCloud will not drop PHP 7.1 for a couple of months, so it will be handy if `dav` continues to advertise PHP 7.1 support, in case there is some patch release needed...)

Reply:
PR #146 will put back PHP 7.1